parlai/agents/memnn/memnn.py

```diff
@@ -57,6 +56,8 @@ def add_cmdline_args(argparser):
         arg_group.add_argument('-histr', '--history-replies', default='label', type=str,
             choices=['none', 'model', 'label'],
             help='Keep replies in the history, or not.')
+        DictionaryAgent.add_cmdline_args(argparser)
```

Comment:
can you add to projects/ models as well?
this could be a problem for other people's models..

Reply:
I found a way to modify the dictionary add_cmdline_args function to resolve so should be good for any other classes now